x11/xorg-server-devel/Portfile

```diff
@@ -88,19 +71,24 @@ configure.args --with-apple-applications-dir=${applications_dir} \
 # GL/internal/dri_interface.h is missing in prefix (provided by libdrm for the xorg DDX... not helpful for us)
 configure.cppflags-append -I/usr/include -I${filespath}/dri -F${configure.sdkroot}/System/Library/Frameworks/ApplicationServices.framework/Frameworks
 
+# Debugging ...
+#configure.cppflags-append  -g -fsanitize=address -fno-omit-frame-pointer
+#configure.cflags-append    -g -fsanitize=address -fno-omit-frame-pointer
+
 configure.env-append \
 	RAWCPP=${configure.cpp}
 
 patchfiles \
-	0001-os-connection-Improve-abstraction-for-launchd-secure.patch \
-	0002-randr-Initialize-RandR-even-if-there-are-currently-n.patch \
-	0003-glx-Initialize-glx-even-if-there-are-currently-no-sc.patch \
-	5000-sdksyms.sh-Use-CPPFLAGS-not-CFLAGS.patch \
-	5001-Revert-dix-Restore-PaintWindow-screen-hook.patch \
-	5002-Workaround-the-GC-clipping-problem-in-miPaintWindow-.patch \
-	5003-Use-old-miTrapezoids-and-miTriangles-routines.patch \
-	5004-fb-Revert-fb-changes-that-broke-XQuartz.patch \
-	5005-fb-Revert-fb-changes-that-broke-XQuartz.patch
+        0001-os-connection-Improve-abstraction-for-launchd-secure.patch \
+        0002-randr-Initialize-RandR-even-if-there-are-currently-n.patch \
+        0003-glx-Initialize-glx-even-if-there-are-currently-no-sc.patch \
+        5000-sdksyms.sh-Use-CPPFLAGS-not-CFLAGS.patch \
+        5001-Revert-dix-Restore-PaintWindow-screen-hook.patch \
+        5002-Workaround-the-GC-clipping-problem-in-miPaintWindow-.patch \
+        5003-Use-old-miTrapezoids-and-miTriangles-routines.patch \
+        5004-fb-Revert-fb-changes-that-broke-XQuartz.patch \
+        5005-fb-Revert-fb-changes-that-broke-XQuartz.patch \
+        5006-work-around-double-delete-on-exit.patch
```

Comment:
I'm surprised none of these patches are in the upstream? (Jeremy usually put stuff in that had been accepted upstream already.)

Reply:
The last one is one I added. It should never go upstream as is as its a work around not a fix.

The others I am not sure, I did not to be honest check if they are upstream or not. They do apply though which suggests not...

Reply:
We might want to investigate why they didn't go upstream then. We'll probably have to do it alone since Jeremy isn't here to explain.